### PR TITLE
Updated program name to just App Assure

### PR DESCRIPTION
**Pull request by ColbyHaase**
Our program name has changed. We are now just App Assure. I removed the work "Desktop" and udated our aka.ms link

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/enterprise/step-1-device-and-app-readiness.md [hunk 1]
```diff
@@ -101,9 +101,9 @@ Configuration Manager software inventory is an alternative to using cloud-based
 For more information, see [Introduction to software inventory in Configuration Manager](https://docs.microsoft.com/configmgr/core/clients/manage/inventory/introduction-to-software-inventory) and setting platform requirements in application packages in [Packages and programs in Configuration Manager](https://docs.microsoft.com/configmgr/apps/deploy-use/packages-and-programs).
 
 
-## Desktop App Assure
+## App Assure
 
-Another tool to help with Windows 10 and Office 365 ProPlus app compatibility is the [Desktop App Assure](https://aka.ms/desktopappassure) program available through the FastTrack Center. Through Desktop App Assure in the event of valid application issues a Microsoft engineer with work with you at no additional cost to help remediate the application incompatibility.
+Another tool to help with Windows 10 and Office 365 ProPlus app compatibility is the [App Assure](https://aka.ms/appassure) program available through the FastTrack Center. Through Desktop App Assure in the event of valid application issues a Microsoft engineer with work with you at no additional cost to help remediate the application incompatibility.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/step-1-device-and-app-readiness.md]: ### 
```suggestion
Another tool to help with Windows 10 and Office 365 ProPlus app compatibility is the [App Assure](https://aka.ms/appassure) program available through the FastTrack Center. In the event of valid application issues, a Microsoft engineer with work with you at no additional cost through App Assure to help remediate the application incompatibility.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@ColbyHaase - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by ColbyHaase** [comment]
@mypil Sorry for the hassle, but I don't use GitHub. Not sure if I did it correctly? The suggested changes are great, so we want to go with those. Please let me know if there's something else I need to do. Thank you!

**Comment by mypil** [comment]
@ColbyHaase - No worries. You did the right thing. I will tag the author now for review and to get his approval to merge this PR. 

Thank you for your contribution to make the docs better! Much appreciated!

**Comment by mypil** [comment]
@JGruszczyk - PR has been copy edited and is ready for final review. 


Thank you.

cc: @kenwith

**Pull request merged**